src/index.js

```diff
@@ -33,9 +29,12 @@ const server = new ApolloServer({
   instrospection: true,
   playground: true,
   tracing: true,
-  context: {
-    models
-  }
+  context: ({ req }) => {
+    const user = req.user
+    if (!user) throw new Error('you must be logged in')
```

Comment:
The job of `context` is to set the context, not to validate if the user is authenticated. This error should not be here. You may want to have some queries/mutations to even non-authenticated users and, with this logic, it won't allow it.